py3status/modules/spotify.py

```diff
@@ -171,6 +191,31 @@ def spotify(self):
         }
         return response
 
+    def on_click(self, event):
+        """
+        Click events
+            - Pause/Play
+            - Next song
+            - Previous song
+
+        Sleep to "invalidate cache" and give the time to change the song to refresh the display
+        """
+
+        button = event['button']
+        if button is None:
+            pass
```

Comment:
I'd get rid of the docstring above and the `if button is None` too as this does not avoid anything.